src/lambda/commands/invokeLambda.ts

```diff
@@ -67,26 +66,22 @@ export async function invokeLambda(params: {
         })
 
         // ideally need to get the client from the explorer, but the context will do for now
-        logger.info('building template...')
-
         const invokeTemplateFn = _.template(LambdaTemplates.INVOKE_TEMPLATE)
-        const resourcePath = path.join(ext.context.extensionPath, 'resources', 'vs-lambda-sample-request-manifest.xml')
 
-        logger.info(sampleRequestManifestPath)
-        logger.info(resourcePath)
+        logger.info('Loading Sample Requests Manifest')
 
         try {
-            const sampleInput = await params.resourceFetcher.getResource([
-                new WebResourceLocation(sampleRequestManifestPath),
-                new FileResourceLocation(resourcePath)
-            ])
-            const inputs: SampleRequest[] = []
+            const sampleInput = await makeSampleRequestManifestResourceFetcher().get()
```

Comment:
Assume this is the call-site clean-up that you were talking about - cos this seems weird to me to create a new one every time.

Reply:
Yes, and you're right, it isn't necessary to create one every time. I think future improvement passes can move in that direction. (I'm thinking of starting with the region loading code instead of the invoke lambda code and introducing a resource provider)